src/toast/toast-config.ts

```diff
@@ -0,0 +1,40 @@
+import {Injectable} from '@angular/core';
+
+export type AriaLiveValue = 'polite' | 'alert';
```

Comment:
Don't see much utility in this type personally.

- it is used only in this file
- it makes documentation more complex

Reply:
I removed it, but I have to duplicate the type info for Typescript to be happy.
For example, I am not keen about `ariaLive: 'polite' | 'alert' = 'polite';` either... but without it does not compile

Reply:
The big upside for me would be the change in this:

![Screen Shot 2019-05-28 at 10 27 22](https://user-images.githubusercontent.com/8074436/58462974-45732500-8133-11e9-9ea7-a996362946f4.png)

It would just say `type: 'polite' | 'alert'`